stonesoup/metricgenerator/ospametric.py

```diff
@@ -27,6 +68,7 @@ class GOSPAMetric(MetricGenerator):
     """
     p: float = Property(doc="1<=p<infty, exponent.")
     c: float = Property(doc="c>0, cutoff distance.")
+    switching_penalty: bool = Property(doc="Penalty term for switching.", default=0.0)
```

Comment:
Is this a `bool` or a `float`? You've hinted `bool` but it looks like a `float` would function in the code. Is this intended? In this case change type hint to `float`. I suspect this is the case as you've called it a loss *factor*.

Reply:
Good catch. I have updated the type hint to float.